packages/lib/actionClass/service.ts

```diff
@@ -69,7 +77,7 @@ export const getActionClassByEnvironmentIdAndName = async (
           select,
         });
 
-        return actionClass;
+        return formatDateFields(actionClass, ZActionClass) as TActionClass;
```

Comment:
is this also working if `formatDateFields()` receives a null value from prisma or do we need a separate check for that?

Reply:
Most of them had null checks before passing value to formatDateFields, and for the rest, I added null checks and also formatDateFields will return null if null value is passed to it now